earth_enterprise/rpms/build.gradle

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 the Open GEE Contributors
+// Copyright 2017, 2018 the Open GEE Contributors
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
```

Comment:
I ran into an issue where it was trying to run rpmrebuild on the ubuntu platform. But, I got around it by only having it build deb packages on Ubuntu. I already had this code ready, but not checked in yet. I think it would be a good idea to go ahead and add that also. 

This would go right after `def rpmPlatformString = org.opengee.os.Platform.rpmPlatformString`
```
def rpmPlatformString = org.opengee.os.Platform.rpmPlatformString
 
def osId = org.opengee.os.Platform.osId

def packageDep

if (osId == "ubuntu") {
    packageDep = "openGeeDebs"
} else {
    packageDep = "openGeeRpms"
}
```

And replace the line `task osPackage(dependsOn: ['openGeeRpms', 'openGeeDebs'])` with `task osPackage(dependsOn: packageDep)`